### ISSUE TITLE
获取联系人的头像异常

### ISSUE THREAD

**Issue by fy2099** [wechaty/python-wechaty-puppet#24]
2020-07-20 17:28:25,638 - Wechaty - ERROR - internal error <(<Status.INTERNAL: 13>, 'FileBox.toJSON() can only work on limited FileBoxType(s). See: <https://github.com/huan/file-box/issues/25>', None)>

            file = await from_contact.avatar()
            await file.to_file(from_contact.contact_id, True)

**Comment by wj-Mcat** [comment]
Yes, this interface is not supported now, it's in our to-do list. If it's fixed, we will mention you later, please keep eyes on this issue. Thanks for your issue.

**Comment by fy2099** [comment]
> Yes, this interface is not supported now, it's in our to-do list. If it's fixed, we will mention you later, please keep eyes on this issue. Thanks for your issue.

还有这个接口看看是不是也有点问题呢
            filebox = await msg.to_file_box()
            await filebox.to_file(filebox.name, True)   #不稳定，经常下载不到图片到本地
            print('图片下载',filebox.name)

2020-07-20 18:28:44,008 - Message - INFO - Message to FileBox
图片下载  5f9740319f0f9eff9519a92dddbe2682.jpg
